zeitschrift-fur-kunstgeschichte.csl

```diff
@@ -47,8 +51,8 @@
     </names>
   </macro>
   <macro name="editor">
-    <names variable="editor translator" delimiter=", ">
-      <name delimiter="" prefix=" " and="text" delimiter-precedes-et-al="never" delimiter-precedes-last="never"/>
+    <names variable="editor translator" delimiter=". ">
```

Comment:
Yeah, you are right, but I would still prefer to have here a comma. When at some point the et-al rule will change, then the style would already be "prepared" for that cases as well.